$OpenBSD: patch-src_test_run-pass_backtrace_rs,v 1.4 2016/12/26 13:52:46 landry Exp $
Skip run-pass/backtrace as libbacktrace is disabled on openbsd.
--- src/test/run-pass/backtrace.rs.orig	Sun Dec 18 00:40:12 2016
+++ src/test/run-pass/backtrace.rs	Wed Dec 21 11:18:23 2016
@@ -103,7 +103,8 @@ fn runtest(me: &str) {
 }
 
 fn main() {
-    if cfg!(windows) && cfg!(target_env = "gnu") {
+    if (cfg!(windows) && cfg!(target_env = "gnu")) ||
+        cfg!(target_os = "openbsd") {
         return
     }
 
